docs/spec/v1beta1/imagerepositories.md

```diff
@@ -77,9 +77,40 @@ For a publicly accessible image repository, you will not need to provide a `secr
 
 When running in [<abbr title="Elastic Kubernetes Service">EKS</abbr>][EKS] and using [<abbr
 title="Elastic Container Registry">ECR</abbr>][ECR] to store images, you should be able to rely on
-the controller retrieving credentials automatically. The controller must be run with the flag
-`--aws-autologin-for-ecr` set for this to work. The advice under "Other platforms" below will also
-work for ECR.
+the controller retrieving credentials automatically.
+
+The `image-reflector-controller` must be run with the flag `--aws-autologin-for-ecr` set for this to work.
+
+This flag can be added by including a patch in the `kustomization.yaml` overlay file in your `flux-system`,
+similar to the process described in [customize Flux manifests][]:
+
+```
+patches:
+  - target:
+      version: v1
+      group: apps
+      kind: Deployment
+      name: image-reflector-controller
+      namespace: flux-system
+    patch: |-
+      - op: add
+        path: /spec/template/spec/containers/0/args/-
+        value: --aws-autologin-for-ecr
+```
+
+Alternatively, the advice under [Other platforms][other platforms] below will also work for ECR.
+
+> You need to upgrade to Flux version 2 release [v0.19][Flux v0.19.0] that contains the image-reflector-controller release [v0.13.0][image-reflector-controller v0.13.0].
```

Comment:
API specs usually don't contain changelogs, what's the reason we add these here?

Reply:
As an experimental feature, I wanted to call out the release version that these docs applied to so that it is clear enough what needs to be updated when changes inevitably happen. (Also, it was a suggestion from @lloydchang)